pkg/flowaggregator/flowaggregator.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flowaggregator
+
+import (
+	"net"
+	"time"
+
+	"github.com/vmware/go-ipfix/pkg/collector"
+	"github.com/vmware/go-ipfix/pkg/entities"
+	"github.com/vmware/go-ipfix/pkg/exporter"
+	"github.com/vmware/go-ipfix/pkg/registry"
+	"k8s.io/klog"
+)
+
+type flowAggregator struct {
+	flowCollectorAddr net.Addr
+	exportFrequency   time.Duration
+}
+
+func InitFlowAggregator(flowCollectorAddr net.Addr, exportFrequency time.Duration) *flowAggregator {
+	registry.LoadRegistry()
+	return &flowAggregator{
+		flowCollectorAddr,
+		exportFrequency,
+	}
+}
+
+func (fae *flowAggregator) Run(stopCh <-chan struct{}) {
+	// Go-ipfix TCP collector doesn't work during my last test, fixed as udp temporarily.
+	collectAddr, _ := net.ResolveUDPAddr("udp", "0.0.0.0:4739")
+	cp, _ := collector.InitCollectingProcess(collectAddr, 1024, 0)
```

Comment:
how is the flow-aggregator collector address fed to the flow exporter at antrea-agent?